### PR TITLE
 #1449: R Toolbar is not visible by default

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

File: src/Package/Impl/Packages/BasePackage.cs [hunk 1]
```diff
@@ -58,6 +61,14 @@ public abstract class BasePackage<TLanguageService> : VisualStudio.Shell.Package
             foreach (var commmand in CreateMenuCommands()) {
                 menuCommandService.AddCommand(commmand);
             }
+
+            var dte = VsAppShell.Current.GetGlobalService<DTE2>(typeof(DTE));
+            var cbs = dte?.CommandBars;
+            var cb = dte?.CommandBars["R Toolbar"] as CommandBar;
```
Comment (AlexanderSher) [inline src/Package/Impl/Packages/BasePackage.cs]: Do we have this string defined anywhere as a constant?

File: src/Package/Impl/Utilities/ViewUtilities.cs [hunk 2]
```diff
@@ -80,9 +80,11 @@ public static class ViewUtilities {
         }
 
         public static void SaveFile(this ITextView textView) {
-            RunningDocumentTable rdt = new RunningDocumentTable(RPackage.Current);
-            string filePath = textView.TextBuffer.GetFilePath();
-            rdt.SaveFileIfDirty(filePath);
+            if (textView != null) {
+                RunningDocumentTable rdt = new RunningDocumentTable(RPackage.Current);
+                string filePath = textView.TextBuffer.GetFilePath();
```
Comment (AlexanderSher) [inline src/Package/Impl/Utilities/ViewUtilities.cs]: With the change above, `filePath` can be `null`. Will `SaveFileIfDirty` handle it?

File: src/Package/Impl/Packages/BasePackage.cs (hunk 1 above)
Comment (MikhailArkhipov) [inline src/Package/Impl/Packages/BasePackage.cs]: in VSCT only so it is not directly importable. VSCT can handle C++ .h files, but that's not for C#.

File: src/Package/Impl/Utilities/ViewUtilities.cs (hunk 2 above)
Comment (MikhailArkhipov) [inline src/Package/Impl/Utilities/ViewUtilities.cs]: Yes

Comment (MikhailArkhipov) [inline src/Package/Impl/Utilities/ViewUtilities.cs]: Actually, let me add another check

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**